linux_cpu_usage.py

```diff
@@ -24,26 +57,33 @@ def get_cpustats_dict(header):
     uptime = 0
     cpustats_dict = {}
     count = 0
+    keys = ('user', 'nice', 'system', 'idle', 'iowait', 'irq', 'softirq')
     for line in stat_data:
         ''' here we have to handle some kind of disk
```

Comment:
I think we should rather use a block comment here, right? If not, then I would suggest it to be docstring, therefore: double quotes.